aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -415,8 +415,15 @@ std::tuple<Tensor&,Tensor&> solve_out(Tensor& solution, Tensor& lu, const Tensor
 
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ inverse ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+/*
+Computes the inverse of n-by-n matrix 'self'
+This is an in-place routine, content of 'self' is overriden.
+'infos' is an int Tensor containing error codes for each matrix in the batched input.
```

Comment:
But there is no `infos` argument to this function? Maybe this line just needs to be merged with the following line describing `infos_lu` and `infos_getri`?